h2o-docs/src/product/data-science/glm.rst

```diff
@@ -1740,7 +1741,41 @@ Regress :math:`z_{i}` on the predictors :math:`x_{i}` using the weights :math:`w
 
 This process is repeated until the estimates :math:`\hat{\beta}` change by less than the specified amount.
 
-**Cost of computation**
+**Tweedie Variance Power Maximum Likelihood**
+
```

Comment:
Just to be clear, I'd add a paragraph here saying that this is used only when we estimate Tweedie variance power, and Tweedie variance power and dispersion.  When we fix the Tweedie variance power we use a simpler formula specified above (in the section where the families are described) unless we estimate dispersion. For dispersion estimation when the Tweedie variance power is fixed we use the Series method.

When we need to estimate the full log likelihood (e.g., for calculating AIC or just the log likelihood) we use the calculation in this section.